src/server/common.h

```diff
@@ -53,8 +53,16 @@ class CommandId;
 class Transaction;
 class EngineShard;
 
+struct LockTagOptions {
+  bool enabled = false;
+  char open_locktag = '{';
+  char close_locktag = '}';
+  unsigned skip_n_end_delimiters = 0;
+  std::string prefix;
+};
+
 struct KeyLockArgs {
-  static bool IsLockHashTagEnabled();
+  static LockTagOptions GetLockTagOptions();
```

Comment:
I suggest keeping `IsLockHashTagEnabled` and move `LockTagOptions` to cc

Reply:
But we need to expose these options for cluster_config, where hash tag extraction is done.
We can move it back, and maybe rename the `KeyTag()` function while on it, but it's kinda beyond the scope here I think.